src/Resources/skeleton/serializer/Normalizer.tpl.php

```diff
@@ -25,6 +25,11 @@ public function normalize($object, $format = null, array $context = array()): ar
 
     public function supportsNormalization($data, $format = null): bool
     {
-        return $data instanceof \App\Entity\BlogPost;
+        // Replace with your own logic
+        // See https://symfony.com/doc/current/serializer/custom_normalizer.html
+        //
+        // return $data instanceof \App\Entity\YourEntity;
```

Comment:
Same note as above, it'd make sense uncommenting this to me.

Reply:
Totally agree, That's what I said in the previous PR.

Reply:
Hmm. I agree. The current example code seems pretty clear, and it's better than needing to know to remove that `return false`. I think we should leave as-is or add a second question to each maker:

> What class will this voter/normalizer handle? (e.g. BlogPost or leave blank)

(if a non-FQN were passed, we'd look for it in `Entity`). If a class is passed, we use that in the generated code. If none is passed, we leave the class-checking part off of the voter and, for the normalizer, probably put a big TODO in `supportsNormalization()`.

WDYT?